content/courses/mobile/intro-to-solana-mobile.md

```diff
@@ -457,36 +472,39 @@ npm run android
 ```
 
 This should open and run the app in your Android emulator. If you run into
-problems, check to make sure you've accomplished everything in the
-[prerequisites section](#0-prerequisites).
+problems, check to make sure you’ve accomplished everything in the
+[_Getting Started_](#getting-started) section.
 
-#### 3. Install Dependencies
+### 3. Install Dependencies
 
-We'll need to add in our Solana dependencies.
+We will need to add in our Solana dependencies.
 [The Solana Mobile docs provide a nice list of packages](https://docs.solanamobile.com/react-native/setup)
 and explanations for why we need them:
 
 - `@solana-mobile/mobile-wallet-adapter-protocol`: A React Native/Javascript API
   enabling interaction with MWA-compatible wallets
 - `@solana-mobile/mobile-wallet-adapter-protocol-web3js`: A convenience wrapper
-  to use common primitives
-  from [@solana/web3.js](https://github.com/solana-labs/solana-web3.js), such
-  as `Transaction` and `Uint8Array`
+  to use common primitives from
+  [@solana/web3.js](https://github.com/solana-labs/solana-web3.js), such as
+  `Transaction` and `Uint8Array`
 - `@solana/web3.js`: Solana Web Library for interacting with the Solana network
-  through the [JSON RPC API](/docs/rpc/http/index.mdx)
-- `react-native-get-random-values` Secure random number generator polyfill
-  for `web3.js` underlying Crypto library on React Native
-- `buffer`: Buffer polyfill; also needed for `web3.js` on React Native
+  through the [JSON RPC API](https://solana.com/docs/rpc)
+- `react-native-get-random-values` Secure random number generator polyfill for
+  <<<<<<< HEAD `web3.js` underlying Crypto library on React Native
+- # `buffer`: Buffer polyfill; also needed for `web3.js` on React Native.
+  `web3.js` underlying library on React Native
+- `buffer`: Buffer polyfill; also needed for `web3.js` on React Native
+  > > > > > > > a3ea04e (minor fixes, resolved conflict)
 
-In addition to this list, we'll add two more packages:
+In addition to this list, we will add two more packages:
 
 - `@coral-xyz/anchor`: The Anchor TS client.
 - `assert`: A polyfill that lets Anchor do its thing.
 - `text-encoding-polyfill`: A polyfill needed to create the `Program` object
 
-If you're not familiar: polyfills actively replace Node-native libraries to make
-them work anywhere Node is not running. We'll finish our polyfill setup shortly.
-For now, install dependencies using the following command:
+If you’re not familiar: polyfills actively replace Node-native libraries to make
```

Comment:
```suggestion
If you’re not familiar: polyfills provide Node-native libraries to make
```

they don't replace node libraries, they provide them. if the node feature exists it won't be polyfilled.

Reply:
my bad, corrected